www/docs/configuration/options.md

```diff
@@ -5,7 +5,7 @@ title: Options
 
 ## Environment Variables
 
-### NEXTAUTH_URL 
+### NEXTAUTH_URL
```

Comment:
Could we revert this file? 

I have a [PR open](https://github.com/nextauthjs/next-auth/pull/1900) updating it extensively and I think it's gonna end a bit in conflict hell 😬 due to the `prettier` changes. I'll add the bit you updated pointing to the `events` documentation  on my PR 👍🏽

Reply:
In fact, NVM I confused the file name 🤦🏽‍♂️ ☕️ , ha ha we're all good 💚